{"uuid":"24d30160-02da-42e5-aba3-c132f7d66d87","historyId":"ab87d85a97d7380422eeb1b84f0da3a7","fullName":"src/test/resources/features/User.feature:68","labels":[{"name":"host","value":"DESKTOP-AKTBA7H"},{"name":"thread","value":"62028@DESKTOP-AKTBA7H.main(1)"},{"name":"feature","value":"Validating User API"},{"name":"story","value":"Check if user able to create with invalidrequestbody"},{"name":"suite","value":"Validating User API"},{"name":"testClass","value":"Check if user able to create with invalidrequestbody"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///C:/Users/rajas/git/repository/UserBDD_Bootcamp/src/test/resources/features/User.feature"},{"name":"package","value":"src.test.resources.features.User_feature.Validating User API"}],"links":[],"name":"Check if user able to create with invalidrequestbody","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected:<...ady exist with same [contact number]> but was:<...ady exist with same [email id]>","trace":"org.junit.ComparisonFailure: expected:<...ady exist with same [contact number]> but was:<...ady exist with same [email id]>\r\n\tat org.junit.Assert.assertEquals(Assert.java:117)\r\n\tat org.junit.Assert.assertEquals(Assert.java:146)\r\n\tat stepDefinition.UserSteps.user_receives_with_message(UserSteps.java:197)\r\n\tat ✽.user receives \"409 CONFLICT\" with message \"User already exist with same email id\"(file:///C:/Users/rajas/git/repository/UserBDD_Bootcamp/src/test/resources/features/User.feature:71)\r\n"},"stage":"finished","steps":[{"name":"Given  user with Invalid request body","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1716501831568,"stop":1716501831571},{"name":"When  user send post Request and request Body with invalid data","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1716501831571,"stop":1716501831990},{"name":"Then  user receives \"409 CONFLICT\" with message \"User already exist with same email id\"","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected:<...ady exist with same [contact number]> but was:<...ady exist with same [email id]>","trace":"org.junit.ComparisonFailure: expected:<...ady exist with same [contact number]> but was:<...ady exist with same [email id]>\r\n\tat org.junit.Assert.assertEquals(Assert.java:117)\r\n\tat org.junit.Assert.assertEquals(Assert.java:146)\r\n\tat stepDefinition.UserSteps.user_receives_with_message(UserSteps.java:197)\r\n\tat ✽.user receives \"409 CONFLICT\" with message \"User already exist with same email id\"(file:///C:/Users/rajas/git/repository/UserBDD_Bootcamp/src/test/resources/features/User.feature:71)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1716501831991,"stop":1716501832023}],"attachments":[],"parameters":[],"start":1716501831567,"stop":1716501832025}